features/0592-indy-attachments/README.md

```diff
@@ -20,51 +20,107 @@ Allows Indy-style credentials to be used with credential-related protocols that
 
 ## Reference
 
-### cred abstract format
+### cred filter format
 
-This format is used to clarify the structure and semantics (but not the concrete data values) of a potential credential -- both in proposals sent from potential holder to issuer, and in offers sent from issuer to potential holder.
+The potential holder uses this format to propose criteria for a potential credential for the issuer to offer.
 
-The identifier for this format is: `hlindy/cred-abstract@v2.0` It is a base64-encoded version of the data returned from [`indy_issuer_create_credential_offer()`](https://github.com/hyperledger/indy-sdk/blob/57dcdae74164d1c7aa06f2cccecaae121cefac25/libindy/src/api/anoncreds.rs#L280).
+The identifier for this format is `hlindy/cred-filter@v2.0`. It is a base64-encoded version of the data structure specifying zero or more criteria from the following (non-base64-encoded) structure:
```

Comment:
It seems odd to me that we'd allow a message to propose criteria, but allow zero criteria. Shouldn't we say "one or more criteria" -- on the theory that if they don't have any criteria to propose, they simply don't propose anything? I guess it depends on whether we think it makes sense for a holder to say, "I propose that you give me a credential, but I'm not going to tell you what schema or cred def or anything else; you can just give me whatever credential you feel like."

This comment is not intended to request a change to the text, so much as just to ask a question. I'm okay merging the PR without it being addressed.

Reply:
Zero or more is just what I implemented - otherwise it needs a check in marshmallow de/serialization to ensure that at least one of these mandatory fields are in place.

Reply:
Okay, thanks for the clarification.